canary-release/cfexample.yaml

```diff
@@ -0,0 +1,17 @@
+title: canary-release
+description: canary release pattern implementation with Codefresh pipelines and K8S
+sourceUrl: http://github.com/Codefresh-Examples/Examples/canary-release
+baseUrl: http://github.com/Codefresh-Examples/Examples/canary-release
```

Comment:
wrong baseurl.. for example http://github.com/Codefresh-Examples/Examples/canary-release/README.md is not a valid url. use github raw url here

Reply:
I can fix this after the PR is merged. Right now there is no actual link to the file in the original repo.